packages/toast/locales/en/messages.po

```diff
@@ -0,0 +1,26 @@
+msgid ""
+msgstr ""
+"POT-Creation-Date: 2024-01-08 14:36+0100\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: @lingui/cli\n"
+"Language: en\n"
+
+#. Default screenreader message for error in toast component
+#. js-lingui-explicit-id
+#: packages/toast/toast.js:112
+msgid "toast.aria.error"
+msgstr "error"
```

Comment:
Shouldn't the strings be with Capital letter?

Reply:
You're right! I'll change it! 😄

Reply:
Updated messages.mjs in this commit: [c2d0332](c2d0332bc19317303a6b58af6a0a0021bf65a1f3)